sources/packages/web/src/services/http/dto/Common.dto.ts

```diff
@@ -16,16 +18,26 @@ export interface InstitutionPrimaryContactAPIOutDTO {
  * a generic address interface that is used extend or used as nested type
  * for both IN and OUT DTOs
  */
-export interface AddressDetailsFormAPIDTO {
+export class AddressDetailsFormAPIDTO {
+  @Expose()
   addressLine1: string;
+  @Expose()
```

Comment:
Do we still need @Expose for optional parameters? Just for my understanding.

Reply:
Just replying here as history since we had that clarify over a call, we need all properties expected to be part of the DTO decorated with `@Expose()` ([more info](https://github.com/typestack/class-transformer#enforcing-type-safe-instance)).